### PR TITLE
build: Add cpack configuration

### PR DESCRIPTION
**Pull request by ictus4u**
Starting block for building project packages using cpack.

### CODE REVIEW

**Review by uristdwarf — COMMENTED** [review]
I'm all for simplifying our Debian package building, but I would like a little more info on what the intention is behind this.

**Review by lgsilva3087 — APPROVED** [review]
LGTM.

File: cmake/Packing.cmake [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+)
+
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/COPYING")
+set(CPACK_RESOURCE_FILE_README "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
+
+set(CPACK_VERBATIM_VARIABLES YES)
+
+set(CPACK_PACKAGE_INSTALL_DIRECTORY ${CPACK_PACKAGE_NAME})
+SET(CPACK_OUTPUT_FILE_PREFIX "${CMAKE_SOURCE_DIR}/_packages")
+#set(CPACK_STRIP_FILES YES)
```
Comment (ralcolea) [inline cmake/Packing.cmake:29]: This "dead code" was left intentionally or we can get riddle of it?

File: cmake/Packing.cmake [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+set(CPACK_COMPONENTS_GROUPING IGNORE)
+
+set(CPACK_DEB_COMPONENT_INSTALL ON)
+set(CPACK_DEBIAN_PACKAGE_VERSION "${PACKAGE_VERSION}")
+set(CPACK_DEBIAN_FILE_NAME "DEB-DEFAULT")
+set(CPACK_DEBIAN_COMPRESSION_TYPE "xz")
+set(CPACK_DEBIAN_DEBUGINFO_PACKAGE ON)
+
+set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
+## TODO(Baldor): Set dependencies for the package
```
Comment (ralcolea) [inline cmake/Packing.cmake:53]: Is this block commented part of some work still in progress?

File: tests/ci_build/run-build.sh [hunk 3]
```diff
@@ -92,4 +94,11 @@ cmake -B "${build_dir}" \
 	"${CMAKE_SAUNAFS_ARGUMENTS[@]}" \
 	"${EXTRA_ARGUMENTS[@]}" "${WORKSPACE}"
 
-nice make -C "${build_dir}" -j "$(nproc)" install
+nice make -C "${build_dir}" -j "$(nproc)"
+if [ "${do_install}" = true ]; then
+	nice make -C "${build_dir}" install
+fi
+
+if [ -f "${build_dir}/CPackConfig.cmake" ]; then
```
Comment (ralcolea) [inline tests/ci_build/run-build.sh:102]: IMHO, it's a good idea to print a message notifying the user whether `CPackConfig.cmake` file is missing.

**Review by ralcolea — APPROVED** [review]
LGTM. I added some minor suggestions/questions, but I think is ready to merge.

**Review by ralcolea — APPROVED** [review]

**Pull request merged**